IntelliTect.TestTools.WindowsTestWrapper/ExampleProjects/Notepad.AutomatedTests/Helpers/NotepadWindowHelpers.cs

```diff
@@ -10,6 +12,37 @@ public class NotepadWindowHelpers
         private readonly NotepadWindow _NotepadWindow = new NotepadWindow();
         private readonly SaveAsWindow _SaveAsWindow = new SaveAsWindow();
 
+        public bool ValidateTextFile(string location, string fileName, string expectedText)
+        {
+            bool comparisonPass = false;
+            string combinedPath = Path.Combine( location, fileName );
+            if ( combinedPath == expectedText )
+            {
+                comparisonPass = true;
+            }
+            return comparisonPass;
+        }
```

Comment:
This method confuses me. 
The entire contents can be reduced to:
```C#
return Path.Combine(location, filename) == expectedText;
```
However if that is what is expected the name of the method and parameters should have their names updated since they seem to imply that it is checking the contents of the file.